setup.py

```diff
@@ -39,14 +39,14 @@
     long_description=long_description,
     keywords='dict, dictionary, mapping, bidirectional, bijection, bijective, injective, two-way, 2-way, double, inverse, reverse',
     url='https://github.com/jab/bidict',
-    license='ISCL',
+    license='ISC',
     packages=['bidict'],
     package_data=dict(bidict=['VERSION']),
     zip_safe=True,
     classifiers=[
         'Development Status :: 4 - Beta',
         'Intended Audience :: Developers',
-        'License :: OSI Approved :: ISC License (ISCL)',
+        'License :: OSI Approved :: ISC License',
```

Comment:
Sure. The second commit has the to the relevant authoritative sources in the extended description:

> see https://opensource.org/licenses/ISC and http://spdx.org/licenses/ISC.html

Sorry that I didn't make this clearer in the PR's main description text. I'll add it now, for future reference.

By the way, I did submit a PR to fix the acronym in the list of classifiers (https://github.com/russianidiot/classifiers.py/pull/1). I'm not entirely sure that is the correct procedure to go about fixing this, but if it isn't, I expect that PR to at least open the conversation to reach that goal.